python/ray/tune/examples/pbt_transformers/pbt_transformers.py

```diff
@@ -92,10 +92,12 @@ def get_model():
         compute_metrics=build_compute_metrics_fn(task_name))
 
     tune_config = {
+        "per_device_train_batch_size": 32,
         "per_device_eval_batch_size": 32,
         "eval_steps": tune.sample_from(
             lambda spec: len(train_dataset) // spec.config["per_device_train_batch_size"] + 1  # noqa: E501
-        ) if not smoke_test else 1,
+        ) if not smoke_test else tune.sample_from(
+            lambda spec: min(1, spec.config["per_device_train_batch_size"])),
```

Comment:
this is a little hard to read; can you instead define a separate function inside this function?

Reply:
Done!